jsonrpc/eth_endpoint.go

```diff
@@ -191,6 +202,57 @@ func (e *Eth) GetBlockTransactionCountByNumber(number BlockNumber) (interface{},
 	return *common.EncodeUint64(uint64(len(block.Transactions))), nil
 }
 
+// GetTransactionByBlockNumberAndIndex returns the transaction for the given block number and index.
+func (e *Eth) GetTransactionByBlockNumberAndIndex(number BlockNumber, index argUint64) (interface{}, error) {
+	num, err := GetNumericBlockNumber(number, e.store)
+	if err != nil {
+		return nil, err
+	}
+
+	block, ok := e.store.GetBlockByNumber(num, true)
+
```

Comment:
Nitpick: remove this empty line.

Reply:
Fixed https://github.com/Ethernal-Tech/blade/pull/144/commits/c758e98c14db22b834fc8504af9c752cbfcec46b